cmd/kubelet/app/server.go

```diff
@@ -643,7 +643,18 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies, featureGate f
 		}
 
 		var reservedSystemCPUs cpuset.CPUSet
+		var staticCPUs cpuset.CPUSet
 		var errParse error
+
+		if s.StaticCPUs != "" {
```

Comment:
I think the idea is that `--static-cpus` is consumed by Static CPU Manager policy, right? Should there rather be a generic way for configuring the CPU Manager policies? For example being able to assign a map of configuration parameters to them, or add a configuration file next to CPU Manager state file?